stackbrew.js

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+
+// Grab last git commit
+function getCommitHasForPath(path) {
+  return require('child_process')
+  .execSync(`git log -1 --format=%H HEAD -- ${path}`)
+  .toString().trim()
+}
+
+const stackbrewPath = path.basename(__filename);
+
+// Header
+let stackbrew = `# this file is generated via https://github.com/nodejs/docker-node/blob/${getCommitHasForPath(stackbrewPath)}/${stackbrewPath}
+
+Maintainers: The Node.js Docker Team <https://github.com/nodejs/docker-node> (@nodejs)
+GitRepo: https://github.com/nodejs/docker-node.git
+GitFetch: refs/heads/main\n`;
+
+// Loop versions
+
+const config = JSON.parse(fs.readFileSync('versions.json', 'utf-8'));
+
+const versions = Object.keys(config).reverse()
+
+const now = new Date().getTime()
+const aplineRE = new RegExp(/alpine*/);
+const slimRE = new RegExp(/\*-slim/);
+
+for(version of versions) {
+  let lts = new Date(config[version].lts).getTime();
```

Comment:
any particular reason these are all `let`s?

Reply:
These are different for each iteration of the loop